doc/source/extended_documentation/regrid/bilinear_land_sea.rst

```diff
@@ -0,0 +1,132 @@
+################################################
+Bilinear regridding with land-sea mask awareness
+################################################
+
+This subpackage provides a fast and unified implementation of bilinear regridding with land-sea mask awareness. It is assumed that the source grid is evenly-spaced in a rectilinear latitude/longitude system. The target grid could be rectilinear or curvilinear. 
```

Comment:
Could you put a sentance here to say what we are aiming to do here. Something along the lines that this ensures that we only use land points from the source grid when calculating values for the target grid land points and vice versa for sea points.

Reply:
Done, thanks